src/app/utils/validators/validators.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { FormControl } from '@angular/forms';
+import { listContainsValidator } from './list-contains.validator';
+
+describe('Validators', () => {
```

Comment:
This describe can be removed.

Reply:
I put it like that in case the other validators need to be tested

Reply:
But we do not have any wrapper for those validators. There are few our own validators and each have it's own file with testing. We can in fact create a class for example:
```
  class SbValidators {
    static listContains() {}
    static someOtherMethod
    ....
  }
```
That way this describe wrapper have more sense. But this is topic for a different discussion.